bitnami/kube-prometheus/Chart.yaml

```diff
@@ -34,4 +34,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-prometheus
   - https://github.com/bitnami/bitnami-docker-alertmanager
   - https://github.com/prometheus-operator/kube-prometheus
-version: 6.0.1
+version: 6.0.2
```

Comment:
Since this is adding a new feature, we should increase the minor version (i.e. `6.0.1` -> `6.1.0`)

Reply:
You are right, it's fixed.